src/style.css

```diff
@@ -0,0 +1,150 @@
+* {
+  box-sizing: border-box;
+}
+
+.page {
+  font-family: "Roboto", sans-serif;
+
+  --color-main-text: #060b35;
+  --color-add-text: #616070;
+  --color-accent: #00acdc;
+  --color-light: #f3f3f3;
+  --main-radius: 5px;
+}
+
+.page__content {
+  margin: 0;
+  padding: 0;
+}
+
+.title {
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  border: 1px solid var(--color-light);
+  border-radius: var(--main-radius);
+}
+
+.card__inner {
+  padding: 32px 16px 15px;
+}
+
+.card__img {
+  width: 159px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__img img {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color-main-text);
+  margin-bottom: 4px;
+}
+
+.card__product {
+  color: var(--color-add-text);
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  margin-bottom: 16px;
+}
+
+.card__star {
+  margin-bottom: 24px;
+}
```

Comment:
I'd recommend you to style your elements in separate .css file like: card.css (for example)